stdlib/arg.mli

```diff
@@ -173,3 +173,19 @@ val current : int ref
     {!Arg.parse} uses the initial value of {!Arg.current} as the index of
     argument 0 (the program name) and starts parsing arguments
     at the next element. *)
+
+val read_arg: string -> string array
+(** [Arg.read_arg file] reads new line separated command line arguments from
```

Comment:
According to your implementation, they are not "new line separated" but "linefeed-separated". What happened to the suggestion of ignoring CR in this case?

Also, this should be `terminated` rather than `separated`.